build-apisix-openresty-asan-centos7.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x
+
+yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
+yum install centos-release-scl -y
+yum -y install gcc gcc-c++ patch wget git make sudo libasan
+yum -y install openresty-openssl111-asan openresty-pcre-asan openresty-zlib-asan
+
+export openssl_prefix=/usr/local/openresty/openssl111
```

Comment:
The prefix should be openresty-asan/openssl111

Reply:
update